client/request/request.go

```diff
@@ -32,6 +67,7 @@ type Directives struct {
 }
 
 type OperationDefinition struct {
+	Name       string
```

Comment:
todo: Sorry for the bother, as the other existing props are not documented, but this is a public member and as such should have some documentation on it.